cluster/store.go

```diff
@@ -64,3 +64,24 @@ func (kvs *KVStore) GetPrefix(ctx context.Context, key string) ([]string, error)
 
 	return gets, nil
 }
+
+// Put sets the value for the given key
+func (kvs *KVStore) Put(ctx context.Context, key, value string) error {
```

Comment:
an idea: with the recent suggestion of the optional options, what do you think about also having the context as apart of options? 

Reason being that it might make the calls simplier when there is no context to be passed. Just a thought
```go
// both work
kvs.Put("hello", "world")
kvs.Put("hello", "world", cluster.WithCtx(ctx))
```